### DISCUSSION TITLE
Shadcn exemple

### DISCUSSION THREAD

**Discussion by dridk** [zauberzeug/nicegui#3711] [Show and tell]
I had fun making 2 design examples that I copied from the [shadcn website.](https://ui.shadcn.com/) I wanted to see if I could reproduce any desgin with nicegui.
Here's what it looks like ( mailbox and Forms ) .  In the end, I had more control using div.element with tailwindcss than with quassar elements. So there's a bit of a mix of the two methods in the code. 

It's not very clean, because I wanted to be quick and I'm not a web expert !

Code source is available here : https://github.com/dridk/nicegui_shadcn 

![image](https://github.com/user-attachments/assets/b01a3ff9-b8cb-44e9-b00a-79458ac24611)

https://ui.shadcn.com/examples/mail
![image](https://github.com/user-attachments/assets/76a06878-4cdc-4450-8aa1-f25b7b5edf57)

**Comment by rodja** [comment]
Cool. Thanks for sharing this!

**Comment by ssizan01** [comment]
This is inspiring. Thank you for this!!
